cypress-E2E-tests/cypress/integration/asset_properties_widget.cy.ts

```diff
@@ -0,0 +1,614 @@
+    ).click();
+    cy.get('.p-l-4 > .dlt-c8y-icon-angle-right').click();
+    cy.selectSubasset(subassetName);
+    cy.get(asset_properties_widget_elements.addPropertyButton).click();
+    // workaround for assettypes cache issue
+    cy.selectAssetProperty('owner');
+    cy.get(asset_properties_widget_elements.selectButton).click();
+    cy.get(asset_properties_widget_elements.addPropertyButton).click();
+    // workaround for assettypes cache issue
+    cy.get(`div[title='${prop}']`).should('exist');
+  });
+
+  // Verfify the presence of load more button if there are more number of assets present in the application
+  it('TC_Asset_Properties_Widget_config_027', () => {
+    cy.get(asset_properties_widget_elements.loadMoreButton).should('exist').click();
+  });
+
+  // Clearing the title field value should result in error
+  it('TC_Asset_Properties_Widget_config_028', () => {
+    cy.get(titleFieldId).clear();
+    cy.get(searchElement).click();
+    cy.get('.form-control-feedback-message > .d-block').should(
+      'contain.text',
+      'This field is required.'
+    );
+  });
+
+  // Verify the search functionality works as expected when there are no matching records
+  it('TC_Asset_Properties_Widget_config_029', () => {
+    const assetName = 'eee';
+    const searchResultsElement = '.legend > span';
+    const strongTextElement = '.c8y-empty-state > .d-flex > .text-medium';
+    const smallTextElement = '.small > small';
+    cy.get(searchElement).type(assetName);
+    cy.get(searchResultsElement).should('have.text', 'Search results');
+    cy.get(strongTextElement).should('have.text', 'No match found.');
+    cy.get(smallTextElement).should('have.text', 'Try to rephrase your search word.');
+  });
+
+  // Verify that on click of an asset in Asset selection section it should navigate to the next screen displaying empty state message
+  it('TC_Asset_Properties_Widget_config_030', () => {
+    const assetName = 'check 3';
+    const backArrowElement = '.dlt-c8y-icon-angle-left';
+    const filterElement = '.dlt-c8y-icon-filter';
+    cy.get(assetsTextElement).should('contain.text', 'Assets');
+    cy.get(asset_properties_widget_elements.loadMoreButton).click();
+    cy.get(`p[title='${assetName}']`).click();
+    cy.get(textElement).should('contains.text', 'Assets > ' + assetName);
+    cy.get(backArrowElement).should('exist');
+    cy.get('input[placeholder="Filter this column…"]').click();
+    cy.get(filterElement).should('exist');
+    cy.get('.c8y-empty-state > .d-flex > .text-medium').should(
+      'have.text',
+      'No results to display.'
+    );
+    cy.get('p.small').should('have.text', 'The selected asset has no children.');
+  });
+
+  // Verify that unchecking the property reflects under Properties section
+  it('TC_Asset_Properties_Widget_config_031', () => {
+    const checkboxSelectedElement = ':nth-child(1) > [style="width: 10%;"] > .ng-untouched';
```

Comment:
In general this type of CSS selector is not suggested. Try to identify it in simpler way.

Reply:
done